<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Professor</class>
 <widget class="QDialog" name="Professor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>71</y>
     <width>411</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cp_name">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cp_username">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cp_pwd">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cp_dept">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Department of Computer Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Mechanical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Environmental Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Life Sciences</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Mathematics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Physics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Chemical Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Geometics Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Civil Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Architecture</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Pharmacy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Biotechnology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Electrical and Electronics Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Department of Management</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="cp_create">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cp_exit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
